components/ActivityForm/index.js

```diff
@@ -0,0 +1,63 @@
+export default function ActivityForm({ onSubmit, activity = {} }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onSubmit(data);
+  }
+
+  return (
+    <form className="form" onSubmit={handleSubmit}>
+      <label htmlFor="title"></label>
```

Comment:
Your form labels need a text otherwise they have no use. Please change this for all `label` elements in this form. And a placeholder is not a replacement for a label. [See this article for more advanced information about inputs.](https://www.ovl.design/text/inclusive-inputs/)